pkg/v1/remote/write_test.go

```diff
@@ -152,18 +160,17 @@ func setupWriterWithServer(server *httptest.Server, repo string, img v1.Image) (
 
 	return &writer{
 		ref:    tag,
-		img:    img,
 		client: http.DefaultClient,
 	}, server, nil
 }
 
-func TestCheckExistingFound(t *testing.T) {
+func TestCheckExistingBlobFound(t *testing.T) {
```

Comment:
Can this be a table driven test with the next couple tests?

Reply:
Good thinking